integration/lifecycle_test.go

```diff
@@ -0,0 +1,136 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go-octopusdeploy/octopusdeploy"
```

Comment:
I had to use this instead of github.com/MattHodge/go-octopusdeploy/octopusdeploy as that ended up not looking my local code

Reply:
The easiest way is to check out the repo into a folder structure matching the repository: 

This is mine `/Users/Matthew/go/src/github.com/MattHodge/go-octopusdeploy`

Update it depending on your `GOPATH`.